app/controllers/templates_controller.rb

```diff
@@ -0,0 +1,96 @@
+class TemplatesController < ApplicationController
+    respond_to :html
+
+    include Seek::IndexPager
+    include Seek::AssetsCommon
+  
+    before_action :templates_enabled?
+    before_action :find_assets, only: [:index]
+    before_action :auth_to_create, only: [:new, :create]
+    before_action :find_and_authorize_requested_item,:only=>[:manage, :manage_update, :show, :edit, :destroy, :update]
+
+    
+  
+    def show
+      respond_to do |format|
+        format.html
+        format.json {render json: @template}  
+      end
+    end
+  
+    def new
+      @tab = 'manual'
+      @template = Template.new
+      respond_with(@template)
+    end
+  
+    def create
+      @template = Template.new(template_params)
+      update_sharing_policies @template
+      @template.contributor = User.current_user.person
+      
+      @tab = 'manual'
+  
+      respond_to do |format|
+        if @template.save
+          format.html { redirect_to @template, notice: 'Template was successfully created.' }
+        else
+          format.html { render action: 'new' }
+        end
+      end
+    end
+
+    def edit
+      respond_with(@template)
+    end
+
+    def update
+      @template.update_attributes(template_params)
+      @template.resolve_inconsistencies
+      respond_to do |format|
+        if @template.save
+          format.html { redirect_to @template, notice: 'Template was successfully updated.' }
+          format.json {render json: @template, include: [params[:include]]}
+        else
+          format.html { render action: 'edit', status: :unprocessable_entity }
+          format.json { render json: @template.errors, status: :unprocessable_entity}
+        end
+      end
+    end
+  
+    def destroy
+      respond_to do |format|
+      if @template.can_delete? && @template.destroy
+        format.html { redirect_to @template,location: templates_path, notice: 'Template was successfully deleted.' }
```

Comment:
hardcoded "Template", but I see later on there is a I18n definition that could be used. Same a couple of lines below

Reply:
This issue and the other similar issues have been solved via the last commit.